test.py

```diff
@@ -72,7 +81,7 @@ def run(wrapper, cmd, check_log=None, extra_wrappers=[]):
             raise Skipped("wrappers not modified")
 
         # if test modified but cannot be run on CI action, raise exception
-        if "GITHUB_ACTION" in os.environ:
+        if WRAPPERS_TO_SKIP_ON_GA and "GITHUB_ACTION" in os.environ:
```

Comment:
`wrapper in WRAPPERS_TO_SKIP_ON_GA`

Reply:
oui, j'ai vu entre temps. corrigé évidemment